lib/_stream_transform.js

```diff
@@ -115,9 +103,16 @@ function Transform(options) {
 
   Duplex.call(this, options);
 
-  this._transformState = new TransformState(this);
-
-  var stream = this;
+  this._transformState = {
+    afterTransform: (er, data) => {
+      return afterTransform(this, er, data);
+    },
```

Comment:
I just copied the state object as-is without really paying attention. I will look into adding `bind()`...